views/home.handlebars

```diff
@@ -1,7 +1,7 @@
 {{!-- Copyright (c) 2020 Red Hat, Inc. --}}
 
-<body class="bx--global-light-ui" aria-label={{title}}>
-    <input type="hidden" id="app-access" value={{props.xsrfToken}}>
+<body aria-label="{{ title }}">
+    <input type="hidden" id="app-access" value="{{props.xsrfToken}}">
```

Comment:
@willkutler Would you please verify this change? I noticed the `arial-label` was getting split up because the lack of quotes, and I figured we didn't need the Carbon class any more, but I wasn't 100% sure whether the changes would break anything...